### PR TITLE
[INFINITY-2836] Fix scheduler DNS name in config file

### PR DESCRIPTION
**Pull request by NimaVaziri**

### CODE REVIEW

File: frameworks/hdfs/docs/kerberos.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ hdfs/data-1-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
 HTTP/data-1-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
 hdfs/data-2-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
 HTTP/data-2-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL
-HTTP/hdfs.marathon.autoip.dcos.thisdcos.directory@LOCAL
```
Comment (elezar) [inline frameworks/hdfs/docs/kerberos.md]: Should we add a link as to how the `hdfs` part changes if we use the non-default name?

File: testing/sdk_utils.py [hunk 2]
```diff
@@ -45,6 +45,11 @@ def get_foldered_name(service_name):
         return service_name
     return '/test/integration/' + service_name
 
+def get_foldered_dns_name(service_name):
```
Comment (elezar) [inline testing/sdk_utils.py]: Why is this in "utils" and not in something else like "hosts"?

File: frameworks/hdfs/tests/test_auth.py [hunk 3]
```diff
@@ -13,6 +13,8 @@
 
 
 log = logging.getLogger(__name__)
+foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
```
Comment (elezar) [inline frameworks/hdfs/tests/test_auth.py]: I would suggest that we not use global variables, and instead use test fixtures if required. I don't feel too strongly about this though.

File: frameworks/hdfs/tests/test_auth.py [hunk 4]
```diff
@@ -46,6 +48,9 @@ def get_principals() -> list:
             )
         )
     principals.extend(config.CLIENT_PRINCIPALS.values())
+
+    service_principal = "HTTP/api.{}.marathon.l4lb.thisdcos.directory".format(foldered_dns_name)
```
Comment (elezar) [inline frameworks/hdfs/tests/test_auth.py]: This is a bit more than a "service" principal though, right?

File: frameworks/hdfs/docs/kerberos.md (hunk 1 above)
Comment (NimaVaziri) [reply frameworks/hdfs/docs/kerberos.md]: Perhaps this is addressed in https://github.com/mesosphere/dcos-commons/pull/1960

**Pull request merged**